pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "Rotary"
+version = "0.3.0"
+description = "Rotary is a snakemake-based scalable pipeline for hybrid assembly of microbial genomes using Nanopore and Illumina data."
+authors = [
+    {name = "Jackson Tsuji"},
```

Comment:
OK to change as follows? : "Jackson M. Tsuji"  (this is the way I usually write my name when publishing)

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/pull/102